INSTALL.md

```diff
@@ -25,17 +25,21 @@ You now have the following things ready:
 
 ### Database setup
 
-#### Use an empty database
+#### Using a prefilled database
 
-1. register via <a href="http://localhost:3000/users/sign_up?" target="_blank">MaMpf-GUI</a>
-2. visit the <a href="http://localhost:1080/" target="_blank">mailcatcher webinterface</a> and confirm your account
-3. make your account admin
-   ```
-   # docker-compose exec mampf rails r "User.first.update(admin: true)"
-   ```
-4. There you go :tada:
+After setting up everything you can call the MaMpf Server on <a href="http://localhost:3000/" target="_blank">localhost:3000</a>. The prepopulated database contains data for several users that you can use to sign in (with the obvious roles). Each of these have `dockermampf` as password.:
+
+- `admin@mampf.edu`
+- `teacher@mampf.edu`
+- `tutor@mampf.edu`
+- `student1@mampf.edu`,..., `student5@mampf.edu`
+
+##### Prefilled database: automatic setup
 
-#### Use a prefilled database
+Just uncomment the lines containing `DB_SQL_PRESEED_URL` and `UPLOADS_PRESEED_URL` in the development docker-compose.yml, start Mampf and you should be good to go after it finished initialising.
+If that for some reason does not work, open a new issue about it (https://github.com/MaMpf-HD/mampf/issues/new) and follow the manual setup instead.
```

Comment:
I do not know if it is clear to all users that if would be best to restore the comment after the first initialisation has been done. Otherwise it will autopopulated each time the container is built, which might not be what is wanted.
Maybe add some short info on this?